static/js/pm_list_data.js

```diff
@@ -0,0 +1,67 @@
+import * as pm_list from "./pm_list";
```

Comment:
We really want to avoid this direction of dependencies.  The `pm_list` module is mostly UI-oriented, and we generally want UI-oriented modules to import data-oriented modules, not vice versa.

The prep commit here would be to extract ` _get_convos()` out `pm_list` into `pm_list_data` and then get test coverage on it.

Reply:
We can go with merging #21423 first and then come over to this as a second preparatory commit for the final version of Collapsible private messages list.